.circleci/config.yml

```diff
@@ -330,13 +330,15 @@ jobs:
       - early-returns
       - build-steps
       - test-steps
+      - persist-artifacts
       - run:
           name: Upload artifacts to S3
           command: |
-            if [[ -n $CIRCLE_BRANCH && "<<parameters.upload>>" == "yes" ]]; then
+            if [[ -n $CIRCLE_BRANCH ]]; then
```

Comment:
Would it be easier to add  `upload: "yes"` when calling the job? (from `upload-artifacts`)

Reply:
Uploaded by default or specifically set to "yes" - so this parameter is redundant.
Decided to cleanup later.